tee-worker/app-libs/stf/src/trusted_call_result.rs

```diff
@@ -100,4 +100,6 @@ pub struct RequestVCResult {
 	// they should be referenced/used only when the client's local IDGraph is empty
 	pub pre_mutated_id_graph: AesOutput,
 	pub pre_id_graph_hash: H256,
+	// can be ignored when it's for one vc request
+	pub batch_vc_idx: u8,
```

Comment:
should we move this to a higher layer like `WorkerRpcReturnValue`? I'm thinking of responses that are not `RequestVCResult` like errors or further batch methods we want to enable